src/Components/home.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const text = 'In this project, I will create a full website for the Math magicians app, consisting of several pages,';
```

Comment:
- [OPTIONAL] I think it could be better if you add more information on your home page, for example, you can explain that this app allows users to do some mathematical operations and you have a `Quote` page that displays quotes randomly 😁 🙃

Reply:
Well resolved ✔️